priv/repo/migrations/20160520141230_add_token_to_member.exs

```diff
@@ -0,0 +1,11 @@
+defmodule Community.Repo.Migrations.AddTokenToMember do
+  use Ecto.Migration
+
+  def change do
+    execute "TRUNCATE TABLE members;"
```

Comment:
That is really cool!

My only issue is if you decide to cycle the secret key, all the users now have no way to edit their things. I'm not sure that's ideal for long term token type things.

I may consider that in a refactoring but this i the pattern I used for jobs so I think for now I'll stick with consistency.